src/main/java/it/gov/pagopa/spontaneouspayment/exception/AppError.java

```diff
@@ -8,8 +8,10 @@
 public enum AppError {
 
     ORGANIZATION_NOT_FOUND(HttpStatus.NOT_FOUND, "Organization not found", "Not found the Organization Fiscal Code %s"),
+    ORGANIZATION_NOT_ENABLED(HttpStatus.UNPROCESSABLE_ENTITY, "Organization not enabled", "The organization is not in suitable state [Organization Fiscal Code = %s; Status = %s]"),
     ORGANIZATION_SERVICE_NOT_FOUND(HttpStatus.NOT_FOUND, "Not found a relation between organization and service", "Not found a relation between Organization Fiscal Code %s and Service Id %s"),
     SERVICE_NOT_FOUND(HttpStatus.NOT_FOUND, "The service configuration was not found", "Not found a service configuration for Service Id %s"),
+    SERVICE_NOT_ENABLED(HttpStatus.UNPROCESSABLE_ENTITY, "Service not enabled", "The service is not in suitable state [Service Id = %s; Status = %s]"),
```

Comment:
`Status = %s` it's not useful, we already  have `"Service not enabled"`

Reply:
Right reporting. Corrected